newsroom/wire/formatters/picture.py

```diff
@@ -0,0 +1,19 @@
+from newsroom.wire.formatters.base import BaseFormatter
+from newsroom.wire.utils import get_picture
+
+
+class PictureFormatter(BaseFormatter):
+
+    MIMETYPE = 'image/jpeg'
+    FILE_EXTENSION = 'jpg'
```

Comment:
afaik they work with png as well, I think there is mimetype on the item, so we will have to make it dynamic probably